app_dart/lib/src/request_handlers/reset_devicelab_task.dart

```diff
@@ -35,8 +35,9 @@ class ResetDevicelabTask extends ApiRequestHandler<Body> {
   Future<Body> post() async {
     checkRequiredParameters(<String>[keyParam]);
     final String encodedKey = requestData[keyParam];
+    final ClientContext clientContext = authContext.clientContext;
     final KeyHelper keyHelper =
-        KeyHelper(applicationContext: context.applicationContext);
+        KeyHelper(applicationContext: clientContext.applicationContext);
```

Comment:
Why was this changed?

Reply:
`context` tries to find `_APPENGINE_CONTEXT`, but test fails in local development.

Reply:
That's interesting. Is this how the other Authenticated RequestHandlers use keyhelper then?

Reply:
I believe so. If not, then we need to update when creating corresponding tests.